README.md

```diff
@@ -140,7 +169,7 @@ would look like for such an application.
 
 Your description:
 ```
-Replace this with your answer
+A RESTful route is the way a user makes requests and interacts with the database performing CRUD (create, remove, update deleted) actions in real time in an application.
```

Comment:
Try to include the word "convention" in your answer, and try leaving out mentions of a database.